src/Holding/ViewHolding/HoldingBoundWith/useBoundWithHoldings.js

```diff
@@ -0,0 +1,21 @@
+import { useQuery } from 'react-query';
+
+import { useOkapiKy } from '@folio/stripes/core';
+
+const useBoundWithHoldings = (boundWithItems) => {
+  const ky = useOkapiKy();
+
+  const queryKey = ['ui-inventory', 'bound-with-holdings', boundWithItems];
```

Comment:
Please use `useNamespace` from stripes-core. Here are a couple of examples of how to use it:

https://github.com/folio-org/ui-inventory/search?q=useNamespace

Reply:
Thanks, that's a useful function.  I had patterned the query off the other two ViewHolding subcomponent helpers useHoldingOrderLines and useReceivingHistory which didn't use useNamepace().  I agree this is a better approach.